scripts/ldev2pcs

```diff
@@ -102,10 +125,29 @@ def configure_location(resource, nodes, fixed_score_string=None):
             score -= 10
 
 def configure():
-    cmd = 'pcs cluster setup --force --local'
-    cmd += ' --name '+cmdline_args.cluster_name
-    cmd += ' '+cmdline_args.mgmt_node
-    run(cmd)
+    # the syntax for making the corosync.conf
+    # file depends on the pacemaker version
+    pacemaker_major_number = get_pacemaker_major_number()
+    if pacemaker_major_number == 1:
+        cmd = 'pcs cluster setup --force --local'
+        cmd += ' --name '+cmdline_args.cluster_name
+        cmd += ' '+cmdline_args.mgmt_node
+    if pacemaker_major_number == 2:
+        cmd = 'pcs cluster setup'
+        cmd += ' '+cmdline_args.cluster_name
+        cmd += ' '+cmdline_args.mgmt_node
+        cmd += ' --corosync_conf /etc/corosync/corosync.conf --overwrite'
+    if not cmdline_args.no_corosync:
+        run(cmd)
```

Comment:
Does this mean that "pcs cluster setup" only creates the corosync file, and does nothing else useful?

Reply:
I believe so, in the man page for pacemaker 2.X it says so for "pcs cluster setup" for "Local only mode" which is what you get when you use "--corosync_conf". It lists a lot of stuff that is not done when "--corosync_conf" flag is used.
The explanation is less detailed for pacemaker 1.X for the "--local" flag, but says:
"--local will only perform changes on the local node".